developer/src/kmc-keyboard-info/test/test-keyboard-info-compiler.ts

```diff
@@ -0,0 +1,51 @@
+import * as fs from 'fs';
+import { assert } from 'chai';
+import 'mocha';
+import { TestCompilerCallbacks } from '@keymanapp/developer-test-helpers';
+import { makePathToFixture } from './helpers/index.js';
+import { KeyboardInfoCompiler } from '../src/index.js';
+import { KmpCompiler } from '@keymanapp/kmc-package';
+
+const callbacks = new TestCompilerCallbacks();
+
+beforeEach(function() {
+  callbacks.clear();
+});
+
+describe('keyboard-info-compiler', function () {
```

Comment:
I'd hope to see a couple more unit tests:

Do we produce .keyboard-Info files for keyboards within a multi-keyboard package?  If so, I imagine that could make for an interesting test.  I gleaned that the package `.kps` file feeds a _lot_ of data into the `.keyboard_info` writing process - it's the source of `options.kmpJsonData` or whatever that field name was.  So, making sure that a multi-keyboard scenario doesn't crash the writer, if such a scenario can reasonably happen, is probably important.

Might be good to have a desktop-only keyboard and a Web-only keyboard case too, if those are reasonably simple to set up.  Just an idea, though... not a requirement.

Reply:
Further unit tests can come later. For now, I already ran this against the entire keyboards repository and got 100% identical results so that's a pretty good outcome given that's the only use of .keyboard_info files.